app/javascript/components/forms/SurveyForm.vue

```diff
@@ -121,48 +123,65 @@ export default {
     },
 
     complete() {
-      this.onComplete(this.survey)
+      this.onComplete(this.survey);
     },
 
     exit() {
-      if (this.dataModel === "Commitment") {
-        this.send(this.survey.data)
+      if (this.dataModel === 'Commitment') {
+        this.send(this.survey.data);
       }
-      Turbolinks.visit("/dashboard")
+      Turbolinks.visit('/dashboard');
     },
 
     nextPage() {
       this.survey.nextPage();
     },
 
     onComplete(sender) {
-      const data = sender.data
+      const data = sender.data;
       if (this.dataModel === 'Commitment') {
         data['state'] = 'live';
       }
-      this.send(data)
+      this.send(data);
+    },
+
+    onAfterRenderQuestion(survey, options) {
+      //Return if there is no description to show in popup
+      if (!options.question.popupdescription) return;
+
+      //Add a button and description div;
+      let btn = document.createElement('button');
+      let description = document.createElement('div')
+      let header = options.htmlElement.querySelector('h5');
+
+      btn.type = 'button';
+      btn.className = 'tooltip trigger';
+      description.className = 'tooltip popup'
+      description.innerHTML = options.question.popupdescription;
+
+      header.appendChild(btn);
+      btn.onclick = () => header.lastChild == description ? header.removeChild(description) : header.appendChild(description);
```

Comment:
Might be good to split this onto multiple lines, and can the `==` be `===` or does it not work?